File: Java Script - Introduction to Programming/2.2. Simple Calculations - Exam Problems/01. Training Lab.js
([width, len]) => ((width * 100 / 120) | 0) * (((len * 100) - 100) / 70 | 0) - 3

//------------------------------------(2)------------------------------

function inputHall([width, len]) {
  let rows = Math.trunc(width * 100 / 120)
  let desk = Math.trunc(((len * 100) - 100) / 70)
  let allPlace = rows * desk - 3
  return allPlace
}

//console.log(inputHall([15, 8.9]))
//console.log(inputHall([8.4, 5.2]))

File: Pyrhon - Introduction to Programming/2.2. Simple Calculations - Exam Problems/01. Training Lab.py
print((float(input()) // 1.2) * ((float(input()) - 1) // 0.7) - 3)

#-------------------------------------(2)-----------------------
from math import trunc

width, leng =float(input()), float(input())
rows = trunc(width * 100 / 120)
desk = trunc(((leng * 100) - 100) / 70)
all_place = rows * desk - 3
print(all_place)

'''
15
8.9
#129
'''
